homeassistant/components/ezviz/camera.py

```diff
@@ -0,0 +1,276 @@
+                    camera_serial,
+                    conf_cameras,
+                )
+
+            async_add_entities(
+                [
+                    EzvizCamera(
+                        hass,
+                        camera_username,
+                        camera_password,
+                        camera_rtsp_stream,
+                        camera["serial"],
+                        camera["name"],
+                        camera["deviceType"],
+                        camera["version"],
+                        camera["status"],
+                        camera["createTime"],
+                        camera["category"],
+                        camera["subCategory"],
+                        camera["customType"],
+                        camera["localIp"],
+                        camera["netIp"],
+                        camera["localRtspPort"],
+                        camera["wanIp"],
+                        camera["netType"],
+                    )
+                ]
+            )
+    except PyEzvizError as exp:
+        _LOGGER.error(exp)
+        return
+
+
+class EzvizCamera(Camera):
+    """An implementation of a Foscam IP camera."""
+
+    def __init__(
+        self,
+        hass,
+        username,
+        password,
+        rtsp_stream,
+        serial,
+        name,
+        device_type,
+        version,
+        status,
+        create_time,
+        category,
+        sub_category,
+        custom_type,
+        local_ip,
+        net_ip,
+        local_rtsp_port,
+        wan_ip,
+        net_type,
+    ):
+        """Initialize an Ezviz camera."""
+        super().__init__()
+
+        self._username = username
+        self._password = password
+        self._rtsp_stream = rtsp_stream
+
+        self._serial = serial
+        self._name = name
+        self._type = device_type
+        self._version = version
+        self._status = status
+        self._create_time = create_time
+        self._category = category
+        self._sub_category = sub_category
+        self._custom_type = custom_type
+        self._local_ip = local_ip
+        self._net_ip = net_ip
+        self._local_rtsp_port = local_rtsp_port
+        self._wan_ip = wan_ip
+        self._net_type = net_type
+
+        self._ffmpeg = hass.data[DATA_FFMPEG]
+
+    @property
+    def should_poll(self) -> bool:
+        """Return True if entity has to be polled for state.
+
+        False if entity pushes its state to HA.
+        """
+        return True
+
+    @property
+    def device_state_attributes(self):
+        """Return the Netatmo-specific camera state attributes."""
+        _LOGGER.debug("Getting new attributes from ezviz camera '%s'", self._name)
+
+        attr = {}
+
+        attr["serial"] = self._serial
+        attr["type"] = self._type
+        attr["version"] = self._version
+        # attr['status'] = self._status
```

Comment:
No commented out code, please.  Fix this or remove it entirely (including the `available` property below).